recipes/cmdline_provenance/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cmdline_provenance" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7abbffd72676c8bed10713790fb46cef44538d709085d6e1e7007d089f4b030d"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - gitpython
+
+test:
+  requires:
+    - git
+  imports:
+    - cmdline_provenance
+
+  commands:
+    python -c "import cmdline_provenance"
```

Comment:
This is the same as the above imports test

Reply:
true thanks have removed